_sass/_components--footer.scss

```diff
@@ -61,3 +64,23 @@ $size--footer-padding-top: 60px;
 .footer-link {
   padding: 0;
 }
+
+// Basically applying same padding left to footer logo
+// than the one being applied to header logo, only
+// showing this tablets and below.
+.footer-logo-small {
+  display: block;
+  position: absolute;
+  left: $space-triple;
+
+  img {
+    // This is the logo image size to match the
+    // header logo.
+    width: 44px;
```

Comment:
so this should be a variable on the variables file don't you think? like "size--brand-logo"